core/controllers/reader.py

```diff
@@ -81,6 +95,22 @@ def test_can_play(self, exploration_id, **kwargs):
 
 def _get_exploration_player_data(
         exploration_id, version, collection_id, can_edit):
+    """Get's exploration data which is returned as a dictionary.
```

Comment:
Get's --> Gets

data --> metadata (for consistency with the rest of the docstring)